### PR TITLE
Missing implementation of AxisAngleTemplate(Vector3)

### PR DESCRIPTION
**Pull request by othlu**
constructor

### CODE REVIEW

File: minkindr/include/kindr/minimal/implementation/angle-axis-inl.h [hunk 1]
```diff
@@ -62,6 +62,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
   // \todo furgalep Check that the matrix was good...
 }
 
+template<typename Scalar>
+AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
+    Scalar norm = angleAxis.norm();
+    CHECK_LE(1e-8, norm);
```
Comment (cfo) [inline minkindr/include/kindr/minimal/implementation/angle-axis-inl.h]: I would turn it around, i.e. `CHECK_GT(norm, 1e-8)`, is it realistic that this never occurs? should we just warn and set a random unit axis?

Comment (othlu) [inline minkindr/include/kindr/minimal/implementation/angle-axis-inl.h]: will turn it around. I would play safe and block the construction as the condition must be handled upstream anyways?

**Comment by cfo** [comment]
LGTM

**Comment by cfo** [comment]
make also a PR into ethz-asl

**Pull request merged**